pkgs/by-name/mo/moneydance/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchurl, makeWrapper, openjdk21, openjfx21, }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "moneydance";
+  version = "2023.3_5064";
+
+  src = fetchurl {
+    url = "https://infinitekind.com/stabledl/${finalAttrs.version}/moneydance-linux.tar.gz";
+    hash = "sha256-liU47bTKGBhDCnEQaKA0rUNPmBYH/zMJ7FOhmt5hQ14=";
+  };
```

Comment:
Use `fetchzip` instead, this will take care of extracting the tarball.

Reply:
Awesome